src/DSharp.Compiler/DSharpStringResources.cs

```diff
@@ -61,5 +61,10 @@ public static string ScriptExportMember(string methodName)
         {
```

Comment:
File remane to make it more generic and use a more specific name space like DSharp.Compiler.Resources

Reply:
rename?